### PR TITLE
fix: temp fix for next-auth peer deps (main)

### PR DESCRIPTION
**Pull request by c-ehrlich**
Closes https://github.com/t3-oss/create-t3-app/issues/504

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

hard codes next-auth version to 4.10

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
So sad right now :( our beta-publish workflow only seem to work on local branches. made #507 to test

**Comment by juliusmarminge** [comment]
Will merge from here so you get the credit in changelog.

Beta version is:

```console
npx create-t3-app@6.0.0-beta.b33f6c1
```

File: cli/src/utils/runPkgManagerInstall.ts [hunk 1]
```diff
@@ -43,6 +43,11 @@ export const runPkgManagerInstall = async (
       } else {
         pkgJson.dependencies![pkgName] = `^${latestVersion.trim()}`; //eslint-disable-line @typescript-eslint/no-non-null-assertion
       }
+
+      // FIXME: temp fix for `next-auth@^4.11` requiring `next@12.2.5` as a peer dependency
+      if (pkg === "next-auth") {
```
Comment (juliusmarminge) [inline cli/src/utils/runPkgManagerInstall.ts:48]: This only fixes noInstall mode. Need to fix regular mode too

**Review by juliusmarminge — CHANGES_REQUESTED** [review]

File: cli/src/utils/runPkgManagerInstall.ts (hunk 1 above)
Comment (c-ehrlich) [reply cli/src/utils/runPkgManagerInstall.ts:48]: should be ok now, the fix i implemented is a bit ugly but hopefully it won't have to stay around for long

Comment (juliusmarminge) [reply cli/src/utils/runPkgManagerInstall.ts:48]: Mind pushing your branch to our repo so we can test?

**Review by nexxeln — APPROVED** [review]
Looks good. Just asking, whats the difference between `@` and `@~`?

**Comment by c-ehrlich** [comment]
> Looks good. Just asking, whats the difference between `@` and `@~`?

@ is the exact version, @~ includes newer patch versions (4.10.x)

**Comment by nexxeln** [comment]
> > Looks good. Just asking, whats the difference between `@` and `@~`?
> 
> @ is the exact version, @~ includes newer patch versions (4.10.x)

Oh okay thanks

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**